src/day8/re_frame/trace/app_state.cljs

```diff
@@ -0,0 +1,54 @@
+(ns day8.re-frame.trace.app-state
+  (:require [reagent.core :as r]
+            [clojure.string :as str]
+            [devtools.formatters.core :as cljs-devtools]
+
+            [cljs.pprint :refer [pprint]]))
+
+(defn string->css [css-string]
+  (->> (str/split (get css-string "style") #";")
```

Comment:
Would be good to put in a docstring for this and show example usage. I think I know what it's doing, but would be better not to guess.